scripts/cli-e2e-test.js

```diff
@@ -92,7 +112,7 @@ async function waitForPageWithText(
   browser,
   path,
   text,
-  { intervalMs = 1000, maxAttempts = 60 } = {},
+  { intervalMs = 1000, maxAttempts = 120 } = {},
```

Comment:
Sure this is needed? I don't think we should accept `yarn start` taking more than a minute, even in CI :p

Reply:
Well it kept failing until i bumped it. Just the creation of the plugin is 30+ seconds alone.

Reply:
Did it fail with the message "Failed to load page ..., max number of attempts reached"?

The timeout isn't started until the plugin creation is complete, so the timeout is just for `yarn start` to pick up the new plugin, build, and start serving it

Reply:
https://github.com/spotify/backstage/runs/516185841?check_suite_focus=true

this is a failed run. it just bails, it seems. if it isn't this timeout, then it's something else - but so far, bumping this number has seemed to fix the problem consistently